src/styles/components/_usage-guidelines.scss

```diff
@@ -1,55 +1,75 @@
-$breakpoint-tablet: 1220px;
+$breakpoint-tablet: 860px;
 
 .cmp-usage-guidelines {
-  background-color: #01f5ac;
-  padding-left: 1.25rem;
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  letter-spacing: 0.02rem;
 
   &__heading {
     position: relative;
     font-family: Viga, sans-serif;
     font-weight: 400;
-    font-size: map.get($font-sizes, "md");
-    padding-top: 3.75rem;
+    font-size: 1.5625rem;
+    margin-top: 0;
   }
 
-  &__content {
-    padding-right: 1.25rem;
-    padding-bottom: 1.25rem;
+  &__image {
+    position: relative;
+    width: 80%;
```

Comment:
I think this image needs to be slightly bigger to match the Figma design. Im not 100% sure though since there's more content in the Figma

Reply:
I bumped up the size in the desktop view (I left this particular line of code the same since it only affects the mobile view). It's still a bit smaller than the figma design, but the quality of the image is pretty low and it's probably best not to stretch it farther.